### PR TITLE
Profile settings keys page

### PR DESCRIPTION
**Pull request by sans-harness**
- Integrates `profile-settings-keys` into `Gitness`, adding support for public keys. 
- Adds navigation from `avatar` on bottom left to `settings/profile`.
- Adds table component for displaying SSH keys in `playgroun/components`.
- Attached screenshot is from `Gitness`. 
Screenshot 2024-10-07 at 1 13 51 PM

### CODE REVIEW

File: apps/gitness/src/App.tsx [hunk 1]
```diff
@@ -247,11 +247,12 @@ export default function App() {
                 },
                 {
                   path: 'general',
-                  element: <SettingsAccountGeneralPage />
+                  element: <SettingsProfileGeneralPage />
                 },
                 {
                   path: 'keys',
-                  element: <SandboxSettingsAccountKeysPage />
+                  // element: <SandboxSettingsAccountKeysPage />
```
Comment (cjlee01) [inline apps/gitness/src/App.tsx:254]: did we need comment?

File: packages/playground/src/pages/mocks/profile-settings/mockKeyList.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { KeysList } from '../../../components/profile-settings-keys-list'
+
+export const data: KeysList[] = [
+  {
+    created: 1728145770582,
+    verified: null,
+    identifier: 'try-1',
+    usage: 'auth',
+    fingerprint: 'SHA256:8H8leG+RViGZYLwzXx4N8htb122oFK2RK8RiMPqCgSk',
```
Comment (cjlee01) [inline packages/playground/src/pages/mocks/profile-settings/mockKeyList.ts:9]: use non PII info?

File: apps/gitness/src/App.tsx (hunk 1 above)
Comment (sans-harness) [reply apps/gitness/src/App.tsx]: Noope my bad, resolved

File: packages/playground/src/pages/mocks/profile-settings/mockKeyList.ts (hunk 2 above)
Comment (sans-harness) [reply packages/playground/src/pages/mocks/profile-settings/mockKeyList.ts:9]: Thats a public key...shouldn't be an issue

**Pull request closed without merging**